kafka-ui-react-app/src/components/Connect/List/BatchActionsBar.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import {Action, ResourceType, ConnectorAction, Connector} from 'generated-sources';
+import useAppParams from 'lib/hooks/useAppParams';
+import { useConfirm } from 'lib/hooks/useConfirm';
+import {clusterConnectorsPath, RouterParamsClusterConnectConnector} from 'lib/paths';
+import {useIsMutating} from '@tanstack/react-query';
```

Comment:
i think the prettier here is not applied.

Reply:
fixed